src/components/AssetDescriptionComponent.jsx

```diff
@@ -8,6 +8,28 @@ import ConfirmAction from './common/ConfirmAction';
 import AssignedTo from './AssignAssetComponent';
 import '../_css/AssetDescriptionComponent.css';
 
+const UNITS = {
+  processor_speed: 'GHz',
+  screen_size: 'inch',
+  storage: 'GB',
+  memory: 'GB'
+};
+
+const getSpecs = (specs) => {
+  if (!specs) {
+    return 'This asset has no specifications';
+  }
+
+  delete specs.id;
+  return Object.entries(specs).map(([label, value]) => {
+    if (!value) {
+      return null;
+    }
+    const specUnit = UNITS[label] || ' ';
+    return <p><span className="asset-specs__label">{`${label.replace(/_/g, ' ')}:`}</span> {`${value} ${specUnit}`}</p>;
```

Comment:
Since this is longer, use parenthesis to wrap it. I.e.
```jsx
return (
  <p>
    <span className="asset-specs__label">{`${label.replace(/_/g, ' ')}:`}</span>
    {`${value} ${specUnit}`}
  </p>
);
```
Don't forget to add empty space after `:` as there is a chance using the parenthesis removed the existing one. If that is the case, then just add the empty space after the colon. I.e.
```
{`${label.replace(/_/g, ' ')}: `}